### PR TITLE
Bump version to 0.2.1

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please rerender

**Comment by wolfv** [comment]
actually we can turn this into a noarch package since we can just remove the "trollius" dependency as python < 3.4 isn't relevant anymore.

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please rerender

**Comment by wolfv** [comment]
We also have this error from the ticker bot which is why this PR wasn't done automatically :)

The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!

Please check the URLs in your recipe with version '8.3.1' to make sure they exist!

We also found the following errors:

 - could not hash URL template 'https://github.com/osrf//{{ name }}/archive/{{ version }}.tar.gz'

**Comment by Tobias-Fischer** [comment]
Ah gosh - any ideas why that is? Looks like a straightforward template to me.

**Comment by wolfv** [comment]
yeah for me, too ... idk.

We could try these new-style github URL instead: 

```
  url: https://github.com/osrf/osrf_pycommon/archive/refs/tags/{{ version }}.tar.gz
```

**Comment by wolfv** [comment]
ah maybe also put the sha256 straight into the yaml instead of in a variable

**Comment by Tobias-Fischer** [comment]
Done

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
 {% set version = "0.2.1" %}
-{% set sha256 = "b369b0d4d81bf3e9f89b38d5b81a957c7e435715e8e9ff06104b975b6e90b119" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://github.com/osrf//{{ name }}/archive/{{ version }}.tar.gz
```
Comment (wolfv) [inline recipe/meta.yaml:11]: there was a double backslash before `//{{ name }}` :)

**Comment by wolfv** [comment]
I was about to push myself :D I saw the double backslash after some squinting

**Pull request merged**